KeyError in line 14 of /net/sgi/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v5/MAIN.smk:
'result_d'
  File "/net/sgi/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v5/MAIN.smk", line 14, in <module>
